.github/workflows/go.yml

```diff
@@ -0,0 +1,12 @@
+name: Docker
```

Comment:
I would rename the file to build.yml as with go we don't really know what is being done

FYI it's missing copyright headers